services/src/main/java/io/grpc/services/HealthStatusManager.java

```diff
@@ -26,6 +26,9 @@
  * created in the constructor of {@code HealthStatusManager}, and it can be retrieved by the
  * {@link #getHealthService()} method.
  * The health status manager can update the health statuses of the server.
+ *
+ * <p>The default, empty-string, service name, {@link #SERVICE_NAME_ALL_SERVICES}, is initialized to
+ *    {@link ServingStatus#SERVING}.
```

Comment:
Remove the extra indentation here as well.

It's part of the style guide. https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs